### DISCUSSION TITLE
How to handle Twilio "stop" messages?

### DISCUSSION THREAD

**Discussion by jordan-brough** [chatwoot/chatwoot#4277] [Q&A] [answered]
We have a Twilio SMS inbox and we get a lot of "stop" messages. Is there a recommended way to handle those so that they don't generate notifications or fill up the inbox?

It feels like an automation rule could work? One that automatically resolves new conversations that are just "stop"?  Except that I don't see a way to set up a rule like that.

**Comment by sojan-official** [comment, marked as answer]
@jordan-brough  We are working on automation rules where you can set conditions based on message content and then further perform conversation actions like `add label` , `change status` 

ref: https://github.com/chatwoot/chatwoot/pull/4033

cc: @tejaswinichile
